src/core/builtins/builtins_files.ml

```diff
@@ -173,14 +173,21 @@ let _ =
       "Return a fresh temporary filename. The temporary file is created empty, \
        with permissions 0o600 (readable and writable only by the file owner)."
     [
+      ( "dir",
```

Comment:
I don't like abbreviations in argument names
```suggestion
      ( "directory",
```

Reply:
Sorry just seeing this. Will apply to `main`.